frontend/apps/reader/modules/readergesture.lua

```diff
@@ -191,10 +193,14 @@ function ReaderGesture:gestureAction(action)
         UIManager:setDirty("all", "full")
     elseif action == "bookmarks" then
         self.ui:handleEvent(Event:new("ShowBookmark"))
-    elseif action == "page_update_up10" then
+    elseif action == "page_jmp_fwd_10" then
         self:pageUpdate(10)
-    elseif action == "page_update_down10" then
+    elseif action == "page_jmp_fwd_1" then
+        self:pageUpdate(1)
```

Comment:
On EPUB this works as expected (at least in page-separated mode) but on PDF/DjVu this specific command definitely isn't the right one. It's supposed to move the display by a screen length, not to literally go to the next page in the document.

Reply:
https://github.com/koreader/koreader/blob/8189945be93b9951a18f6de9a3b7b9b0cd97fb7a/frontend/apps/reader/modules/readerrolling.lua#L393-L401

Reply:
Stupid idea: send a "TapForward" event instead?

Reply:
Doesn't sound that stupid. ReaderPaging also does one that' called "Rel", just a different one.

```
function ReaderPaging:onTapForward()
    self:onPagingRel(1)
    return true
end

function ReaderPaging:onTapBackward()
    self:onPagingRel(-1)
    return true
end
```